openapi.json

```diff
@@ -5152,6 +5169,14 @@
             "description": "Optionally, a list of CPU numbers for CPUs to which the pipeline may pin\nits worker threads.  Specify at least twice as many CPU numbers as\nworkers.  CPUs are generally numbered starting from 0.  The pipeline\nmight not be able to honor CPU pinning requests.\n\nCPU pinning can make pipelines run faster and perform more consistently,\nas long as different pipelines running on the same machine are pinned to\ndifferent CPUs.",
             "default": []
           },
+          "provisioning_timeout_s": {
```

Comment:
_seconds.

Reply:
Changed to `_secs` to keep consistent with other existing runtime configuration parameters.